### PR TITLE
BDDMockito: rename willNothing to willDoNothing

### PR DESCRIPTION
**Pull request by alb-i986**
.. and keep the original signature as deprecated

### CODE REVIEW

File: src/main/java/org/mockito/BDDMockito.java [hunk 1]
```diff
@@ -346,10 +346,18 @@ public void shouldHaveNoMoreInteractions() {
...
          * See original {@link Stubber#doNothing()}
          * @since 1.8.0
+         * @deprecated please use {@link #willDoNothing()} instead
          */
+        @Deprecated
         BDDStubber willNothing();
 
         /**
+         * See original {@link Stubber#doNothing()}
+         * @since 1.10.20
```
Comment (alb-i986) [inline src/main/java/org/mockito/BDDMockito.java]: Please notice this line.
I think this change should go in 1.x releases.
Or please let me know what version I should enter.

**Comment by szpak** [comment]
It linguistically sounds better to me.

I wonder if we need regression tests to stay in place if original method will be removed one day.

Btw, @alb-i986 Travis failed, but it looks like a flaky test. Could you trigger it once again (e.g. with `git commit --amend`).

**Comment by alb-i986** [comment]
> I wonder if we need regression tests to stay in place if original method will be removed one day.

Looks like there are no tests in place for `willNothing`. Probably because it is not doing much but just forwarding to `Mockito#doNothing`.

**Comment by alb-i986** [comment]
What's the story with this?
Maybe you want me to first open an issue as per contributing guidelines?
Cheers

**Comment by TimvdLippe** [comment]
@alb-i986 thanks for the ping. I think this is fine, so LGTM. The deprecated method can be removed in Mockito 3.0.

**Pull request merged**